### ISSUE TITLE
2.1.0 - New / Edit Product interface is missing product info section (name, sku, enabled, etc.)

### ISSUE THREAD

**Issue by itdint** [magento/magento2#6343]
When clicking on products > catalog > and new / or edit product, the interface is broken.  I can't find a good reference of what the 2.1 product entry interface looks like - i know the sidebar went away, but it appears to be missing the first accordion section that has name, sku, enabled, etc.  You cannot add a new product because these fields do not exist and it will not pass verification. I have searched the issues pages and in general for several days and have not found this particular issue addressed.
### Preconditions
1. Magento 2.1.0
2. Server is Ubuntu 14.04, nginx 1.4.6 php 7.0.9, MySQL 5.0.12
3. Site was installed via composer - upgraded from 2.0.4, have applied all patches.
4. I am running a custom front-end theme (Porto 2.3.2) however, it does not alter any adminhtml files.
5. I am using the nginx.conf exactly as it was provided in the magento 2.1 downloaded release.
6. php env vars have been updated - memory_limit is 1048M just to make sure that isn't an issue.
### Steps to reproduce
1. I upgraded via commandline / composer from 2.0.4 to 2.1.0.  Nothing out of the ordinary.
2. Error happens regardless of deployment mode, or whether I re-compile, or re-deploy static content.
3. So far - all other admin areas, config panels, seem to work correctly.
### Expected result
1. I should be able to set the name, sku, etc. of a product.  As of now I can't add new products, or successfully edit existing products.  
### Actual result
1. Logs - I have scanned nginx logs, syslogs, magento's debug and exception logs - no errors are present that I can find.
2. I have monitored the page in the inspector console - there are no indications of missing js files or any js errors.  No console output at all.

![mage21_product_issue](https://cloud.githubusercontent.com/assets/5232911/17976921/8d8b8fd8-6abd-11e6-9bb2-6d4e92346242.jpg)

**Comment by iazel** [comment]
Hi, can you give us the list on enabled modules? And more importantly, have you applied the patch provided by Porto for magento 2.1?
You should have a `Porto Theme(Magento 2.1 Patch).zip` file.

**Comment by itdint** [comment]
Well - it may not matter.  I think whatever issue this is, it has to do with the upgrade from 2.0.4 to 2.1.0.  I created a clean 2.1.0 install on the same server - thinking I could use it for code comparison, and the clean install works perfectly.  I installed the same porto theme, patches, etc.. and it still worked - so I just moved the catalog from the broken store to the new one.  The strange thing is there was never any error - logged or in the console.  Made it really hard to troubleshoot.

I'll go ahead and close this since i found a workaround.
